Makefile

```diff
@@ -1,13 +1,61 @@
-BINDIR?= bin
-PLATFORM?=linux
-ARCH?=amd64
+all: build
 
-GO_BUILD= env GOOS=$(PLATFORM) GOARCH=$(ARCH) go build -i $(GOFLAGS) \
+# Define constants
+##################
+BINDIR        ?= bin
+PLATFORM      ?= linux
+ARCH          ?= amd64
+CLUSTERAPI_BIN = $(BINDIR)/cluster-api
+VERSION       ?= $(shell git describe --always --abbrev=7 --dirty)
+GO_VERSION     = 1.9
```

Comment:
Any reason why we can't default to 1.10

Reply:
I was a copy-paste from cluster-operator

Reply:
I know, so the question still stands :)

Reply:
I don't see any reason.